docs/en/framework/ui/blazor/branding.md

```diff
@@ -24,11 +24,11 @@ namespace MyCompanyName.MyProjectName.Blazor
 }
 ````
 
-> Currently, setting the `AppName` is only applicable to the [Basic Theme](../../Themes/Basic.md), it does not have any effect on the other [official themes](../../Themes/Index.md).
+> Currently, setting the `AppName` is only applicable to the [Basic Theme](../../Themes/Basic.md), it does not have any effect on the other [official themes](../../../ui-themes).
```

Comment:
Should be `(./basic-theme.md)`

Reply:
I did not change it because no such file was found.

Reply:
The file was moved to `./basic-theme.md`
<img width="403" alt="image" src="https://github.com/abpframework/abp/assets/16813853/3089a69b-e865-4069-b268-2d0869d363e6">